libres/lib/analysis/update.cpp

```diff
@@ -145,10 +129,10 @@ void serialize_node(const char *node_key, const active_list_type *active_list,
    The return value is the number of rows in the serialized
```

Comment:
remove comment, as the function does not return anything anymore

Reply:
Fixed.